src/FreeRTOSConfig.h

```diff
@@ -62,7 +62,7 @@
 #define configTICK_RATE_HZ                      1024
 #define configMAX_PRIORITIES                    (3)
 #define configMINIMAL_STACK_SIZE                (120)
-#define configTOTAL_HEAP_SIZE                   (1024 * 17)
+#define configTOTAL_HEAP_SIZE                   (1024 * 40)
```

Comment:
That unallocated memory is essentially wasted right? So can we make the heap automatically use all available space?

Reply:
```
Memory region         Used Size  Region Size  %age Used
           FLASH:      406164 B     474632 B     85.57%
     SPARE_SPACE:          0 GB        12 KB      0.00%
             RAM:       63720 B        64 KB     97.23%
```

In this case, 63720B + 1024B (for the main stack) are allocated, which leaves 792B un-allocated space which are basically wasted (not used by the application). 

So yes, that would be really nice if we could make the heap automatically use all the available space. Not sure how to do this tough...